src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/VariantAnnotationArgumentCollection.java

```diff
@@ -0,0 +1,57 @@
+package org.broadinstitute.hellbender.cmdline.argumentcollections;
+
+import org.broadinstitute.barclay.argparser.Advanced;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.utils.Utils;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Arguments for requesting VariantContext annotations to be processed by {@link org.broadinstitute.hellbender.tools.walkers.annotator.VariantAnnotatorEngine}
+ * for tools that process VariantContext objects.
+ */
+public class VariantAnnotationArgumentCollection implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    /**
+     * Argument collection constructor which defines tool default annotation arguments when they are not overridden by the user
+     *
+     * @param defaultGroups                      List annotation groups names to be used by default
+     * @param defaultAnnotations                 List of annotation class names to be used by default
+     * @param defaultAnnotationsToExclude        List of default annotation classes to exclude from output, these override the previous two fields
+     */
+    public VariantAnnotationArgumentCollection(List<String> defaultGroups, List<String> defaultAnnotations, List<String> defaultAnnotationsToExclude) {
+        Utils.nonNull(defaultGroups);
+        Utils.nonNull(defaultAnnotations);
+        Utils.nonNull(defaultAnnotationsToExclude);
+
+        annotationGroupsToUse = new ArrayList<>(defaultGroups);
+        annotationsToUse = new ArrayList<>(defaultAnnotations);
+        annotationsToExclude = new ArrayList<>(defaultAnnotationsToExclude);
+    }
+
+    /**
+     * Which annotations to include in variant calls in the output.
```

Comment:
Add note that these supplement the annotations in the annotation groups specified via -G

Reply:
done